openquake.cfg

```diff
@@ -41,7 +41,8 @@ celery_queue = celery
 [dbserver]
 # enable multi_user if you have a multiple user installation
 multi_user = false
-file = ~/db.sqlite3
+file = ~/oqdata/db.sqlite3
+log = ~/oqdata/dbserver.log
```

Comment:
Now both oq-engine and upgrade_manager are creating the datastore.DATADIR if not already there; the tests were broken because I was not passing the log file to the dbserver.